packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -83,7 +83,7 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         'onUri',
         'onWebviewPanel',
         'onFileSystem'
-    ]);
+    ].map(e => e.toLowerCase()));
```

Comment:
let's check with VS Code whether we should normalize to lower case during activation as well

It is because of bogus activation event in the python extension: https://github.com/microsoft/vscode-python/blob/master/package.json#L66

Reply:
nope, i will revert it and fail a bug for python

Reply:
fix for the python extension: https://github.com/microsoft/vscode-python/pull/13468